extensions/resource-deployment/src/ui/modelViewUtils.ts

```diff
@@ -1646,7 +1651,11 @@ export function getPasswordMismatchMessage(fieldName: string): string {
 export async function setModelValues(inputComponents: InputComponents, model: Model): Promise<void> {
 	await Promise.all(Object.keys(inputComponents).map(async key => {
 		const value = await inputComponents[key].getValue();
-		model.setPropertyValue(key, value);
+		if (typeof (value) === 'object') {
```

Comment:
Add a comment here explaining what this check is for (specifically why do we only set the property value to the name for CategoryValues)